src/Adapter/Cart/CommandHandler/UpdateProductQuantityInCartHandler.php

```diff
@@ -129,7 +129,7 @@ private function updateProductQuantityInCart(Cart $cart, UpdateProductQuantityIn
         // It seems that $updateResult can be -1,
         // when adding product with less quantity than minimum required.
         if ($updateResult < 0) {
-            $minQuantity = $command->getCustomizationId() ?
+            $minQuantity = $command->getCombinationId() ?
```

Comment:
```suggestion
            $minQuantity = $combinationIdValue ?
```

Haha such a shitty bug 😅. Using the already present variable will reduce potential bugs even more